snakemd/generator.py

```diff
@@ -1226,23 +1232,27 @@ def add_table(
                     ["1st", "Robert"],
                     ["2nd", "Rae"]
                 ],
-                [Table.Align.CENTER, Table.Align.RIGHT]
+                [Table.Align.CENTER, Table.Align.RIGHT],
+                0
             )
 
         .. versionchanged:: 0.2.0
             Returns Table generated by this method instead of None. 
         .. versionchanged:: 0.4.0
             Added optional alignment parameter
+        '' versionchanged:: x.x.x
```

Comment:
```suggestion
        '' versionchanged:: 0.11.0
```

Reply:
Great! I changed `x.x.x` to `0.11.0` and also removed trailing whitespaces in inline docs (actually Vim and VSCode did).